src/Files.Uwp/Views/SettingsPages/Preferences.xaml

```diff
@@ -195,16 +195,6 @@
                                 Style="{StaticResource RightAlignedToggleSwitchStyle}" />
                         </local:SettingsBlockControl>
 
-                        <local:SettingsBlockControl Title="{helpers:ResourceString Name=SettingsListAndSortDirectoriesAlongsideFiles}" HorizontalAlignment="Stretch">
-                            <local:SettingsBlockControl.Icon>
-                                <FontIcon Glyph="&#xE174;" />
-                            </local:SettingsBlockControl.Icon>
-                            <ToggleSwitch
-                                AutomationProperties.Name="{helpers:ResourceString Name=SettingsListAndSortDirectoriesAlongsideFiles}"
-                                IsOn="{Binding ListAndSortDirectoriesAlongsideFiles, Mode=TwoWay}"
-                                Style="{StaticResource RightAlignedToggleSwitchStyle}" />
-                        </local:SettingsBlockControl>
```

Comment:
It's up for discussion but I think we should keep this setting to configure the default behavior.

Reply:
The idea would be to just set the default value, if the user wants it to apply to all folders, they can adjust the individual folder setting.